tests/common.bash

```diff
@@ -368,7 +368,9 @@ function restart_crio_service() {
 # Configures containerd
 function overwrite_containerd_config() {
 	containerd_config="/etc/containerd/config.toml"
+	base_config_dir=$(dirname "${containerd_config}")
 	sudo rm -f "${containerd_config}"
```

Comment:
The patch is correct but this `rm -f ` is suspicious... it feels weird to remove a file from a directory that might not exist and it isn't actually needed since the default behavior of `tee` is to overwrite pre-existing files. Can you go a bit further and drop that line please ?